From: =?utf-8?q?Jeremy_B=C3=ADcha?= <jeremy.bicha@canonical.com>
Date: Wed, 6 Mar 2024 16:44:28 -0500
Subject: Revert "about-window: add zero-clause bsd license"

This reverts commit d56fb6ab95a04969df63399df85ce13ef1673f24.

It requires gtk4 4.13
Forwarded: not-needed
---
 src/adw-about-window.c | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/src/adw-about-window.c b/src/adw-about-window.c
index 6a81617..0059982 100644
--- a/src/adw-about-window.c
+++ b/src/adw-about-window.c
@@ -218,13 +218,12 @@ static const LicenseInfo gtk_license_info [] = {
   { N_("GNU Affero General Public License, version 3 only"), "https://www.gnu.org/licenses/agpl-3.0.html", "AGPL-3.0-only" },
   { N_("BSD 3-Clause License"), "https://opensource.org/licenses/BSD-3-Clause", "BSD-3-Clause" },
   { N_("Apache License, Version 2.0"), "https://opensource.org/licenses/Apache-2.0", "Apache-2.0" },
-  { N_("Mozilla Public License 2.0"), "https://opensource.org/licenses/MPL-2.0", "MPL-2.0" },
-  { N_("BSD Zero-Clause License"), "https://opensource.org/license/0bsd", "0BSD" }
+  { N_("Mozilla Public License 2.0"), "https://opensource.org/licenses/MPL-2.0", "MPL-2.0" }
 };
 /* Copied from GTK 4 for consistency with GtkAboutDialog. */
 /* Keep this static assertion updated with the last element of the enumeration,
  * and make sure it matches the last element of the array. */
-G_STATIC_ASSERT (G_N_ELEMENTS (gtk_license_info) - 1 == GTK_LICENSE_0BSD);
+G_STATIC_ASSERT (G_N_ELEMENTS (gtk_license_info) - 1 == GTK_LICENSE_MPL_2_0);
 
 typedef struct {
   char *name;
